dash/dash.py

```diff
@@ -378,15 +379,25 @@ def _relative_url_path(relative_package_path='', namespace=''):
 
         srcs = []
         for resource in resources:
+            is_dynamic_resource = 'dynamic' in resource and \
+                resource.get('dynamic')
+
             if 'relative_package_path' in resource:
                 if isinstance(resource['relative_package_path'], str):
```

Comment:
An alternate way might solve the too-many-branches :camel:
```
paths = resource['relative_package_path']
paths = [paths] if isinstance(paths, str) else paths
for rel_path in paths:
   ... the logic that is repeated twice ...

```

Reply:
Sure.. I'm in, if only because it makes me learn the language haha!